deepfence_ui/app/scripts/constants/action-types.js

```diff
@@ -201,22 +201,25 @@ const ACTION_TYPES = [
   'TOP_VULNERABLE_NODES_REQUEST',
   'TOP_VULNERABLE_NODES_SUCCESS',
   'TOP_VULNERABLE_NODES_FAILURE',
+  'REPORT_GENERATION_REQUEST',
+  'REPORT_GENERATION_SUCCESS',
+  'REPORT_GENERATION_FAILURE',
+  'REPORT_STATUS_REQUEST',
+  'REPORT_STATUS_SUCCESS',
+  'REPORT_STATUS_FAILURE',
+  'DOWNLOAD_REPORT_REQUEST',
+  'DOWNLOAD_REPORT_SUCCESS',
+  'DOWNLOAD_REPORT_FAILURE',
+  'REPORT_EMAIL_SCHEDULE_REQUEST',
+  'REPORT_EMAIL_SCHEDULE_SUCCESS',
+  'REPORT_EMAIL_SCHEDULE_FAILURE',
   'XLSX_REPORT_DOWNLOAD_REQUEST',
   'XLSX_REPORT_DOWNLOAD_SUCCESS',
   'XLSX_REPORT_DOWNLOAD_FAILURE',
   'XLSX_EMAIL_SCHEDULE_REQUEST',
   'XLSX_EMAIL_SCHEDULE_SUCCESS',
   'XLSX_EMAIL_SCHEDULE_FAILURE',
```

Comment:
Also these?

Reply:
Similarly, we are using these actions to download XLSX reports on the vulnerabilities page.